release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -131,6 +131,13 @@ When working in a highly regulated environment, you might need the ability to se
 
 For more information, see xref:../networking/dns-operator.adoc#nw-dns-forward_dns-operator[Using DNS forwarding].
 
+[id="ocp-4-11-ovn-internal-traffic-policy-support"]
+==== Internal traffic support for OVN-Kubernetes
+
+As a cluster administrator, you can configure `internalTrafficPolicy=Local` on a Kubernetes service object when using the OVN-Kubernetes Container Network Interface (CNI) cluster network provider. This feature allows cluster administrators to route traffic to an endpoint on the same node that the traffic originated from. If there are no local node endpoints traffic will be dropped.
```

Comment:
I suggest adding a comma between 'endpoints' and 'traffic'. [ISG page 90 - https://source.redhat.com/groups/public/ccs/ccs_folder/ibm_style_guide/ibm_stylepdf]